MatrixSDK/Data/MXRoomLastMessage.m

```diff
@@ -58,6 +59,19 @@ - (instancetype)initWithEvent:(MXEvent *)event
     return self;
 }
 
+- (nullable NSData*)encryptedAttributedString
+{
+    if (_attributedText)
```

Comment:
If not within a `init` method it is best to use synthesized accessors instad of ivars, i.e. `self.attributedText`